src/viam/sdk/robot/client.cpp

```diff
@@ -37,18 +37,8 @@
 namespace viam {
 namespace sdk {
 
-// TODO(RSDK-4573) Having all these proto types exposed here in the APIs is sad. Let's fix that.
-
 using google::protobuf::RepeatedPtrField;
-using viam::common::v1::PoseInFrame;
-using viam::common::v1::ResourceName;
-using viam::common::v1::Transform;
-using viam::robot::v1::Discovery;
-using viam::robot::v1::DiscoveryQuery;
-using viam::robot::v1::FrameSystemConfig;
-using viam::robot::v1::Operation;
 using viam::robot::v1::RobotService;
-using viam::robot::v1::Status;
```

Comment:
Not a big deal either way, but I think maintaining these `using` lines in the `.cpp` file (as opposed to the `.hpp`) is actually fine and may be worth doing if typing out the fully qualified type becomes tedious.